app/views/entities/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1 class="is-size-4 mb-3 has-text-grey is-flex is-justify-content-center has-text-weight-bold">Transaction Details</h1>
+
+<div class="container" style="width: 90%;">
+<p class="is-size-5 is-italic has-text-centered my-3 has-text-weight-bold has-text-grey"><%= @group.name %></p>
```

Comment:
- [ ] Dear @Theodoraldo, According to the project requirements, we need to validate all the input fields form the user. This way we can protect our app form unexpected crashes and can save user from making silly mistakes. currently when I try to register a transaction without feeling the form the application crashes and I get an exception. Instead, we can make the input fields required from the HTML and we can also handle such invalid inputs from the backend and protect our app from crashing. 

![Screen Shot 2023-10-19 at 11 44 58 PM](https://github.com/Theodoraldo/Budget_app/assets/69077061/60c9a2c3-7d69-442c-b44d-1a72ce625154)